packages/web/docs/src/content/gateway/deployment/node-frameworks/nestjs.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description:
+  Nest (Nest JS) is a progressive Node.js framework for building efficient, reliable and scalable
+  server-side applications.
+---
+
+import { Callout } from '@theguild/components'
+
+# Integration with NestJS
+
+[Nest (Nest JS)](https://nestjs.com) is a progressive Node.js framework for building efficient,
+reliable and scalable server-side applications.
+
+Hive Gateway provides its own Nest GraphQL Driver that support building
+[GraphQL Federation](https://the-guild.dev/graphql/hive/federation) gateways including
+[subscription support](/docs/gateway/subscriptions) that do not exist in the default Nest Federation
+drivers.
+
+<Callout>
+  For the setup of a new Nest project, please make sure to read the [Nest GraphQL
+  documentation](https://docs.nestjs.com/graphql/quick-start).
+</Callout>
+
+## Install
+
+```sh npm2yarn
+npm i @nestjs/graphql @graphql-hive/nestjs graphql
+```
+
+## Create Application Module
+
+```typescript
+import { HiveGatewayDriver, HiveGatewayDriverConfig } from '@graphql-hive/nestjs'
+import { Module } from '@nestjs/common'
+import { GraphQLModule } from '@nestjs/graphql'
+
+@Module({
+  imports: [
+    GraphQLModule.forRoot<HiveGatewayDriverConfig>({
+      driver: HiveGatewayDriver,
+      // All configuration for Hive Gateway comes here;
+      supergraph: './supergraph.graphql',
+      // Install subscription handlers to the server
+      installSubscriptionHandlers: true,
+      subscriptions: {
+        // Enable WebSocket subscriptions
+        'graphql-ws': true
+      }
+    })
+  ]
+})
+export class AppModule {}
+```
+
+<Callout>
+  Learn more about subscriptions in Hive Gateway [here](/docs/gateway/subscriptions).
+</Callout>
+
+## Further development
+
+Hive Gateway offers just a gateway driver; meaning, everything else works as
+[showcased in NestJS federation documentation](https://docs.nestjs.com/graphql/federation).
```

Comment:
You use NestJS and Nest interchangeably, let's decide to use one. Nest alights with how the Nest team [calls it](https://docs.nestjs.com/).

```suggestion
[showcased in Nest federation documentation](https://docs.nestjs.com/graphql/federation).
```